index.html

```diff
@@ -7,50 +7,219 @@
+    <link rel="stylesheet" href="./style/dark-style.css"> 
+    <link rel="stylesheet" href="./style/light-style.css"> 
 
     <!--Roboto-->
     <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
     <!--Crete round-->
     <link href='https://fonts.googleapis.com/css?family=Crete%20Round' rel='stylesheet'>
-    <!--Crete round-->
+    <!--Poppins-->
     <link href='https://fonts.googleapis.com/css2?family=Poppins' rel='stylesheet'>
+    <!--Inter-->
+    <link href='https://fonts.googleapis.com/css2?family=Inter' rel='stylesheet'>
 
     <!-- Add icon library -->
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-
 </head>
 <body>
-    <header>
+    <!--Main Mobile-->
+    <div class="dark">
+        <header>
+
+            <nav>
+                <ul>
+                    <li >
+                        <a class="my-logo" href="#">Alex Dev</a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="white" class="bi bi-list" viewBox="0 0 16 16">
+                                <path fill-rule="evenodd" d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"/>
+                            </svg>
+                        </a>
+                    </li>
+                </ul> 
+            </nav>
+
+        </header>
+        <section>
+            <h1>Hey There. I'm Alex</h1>
+            <h2>I'm a Software Developer</h2>
+            <p>I can help you build a product, feature or website Look trough some of my work and experience! If you like what you see and have a project you need coded, don't heastiate contact me.</p>
+        </section>
+        <footer>
+            <a href="#" class="fa fa-github fa-lg"></a>
+            <a href="#" class="fa fa-linkedin fa-lg"></a>
+            <a href="#" class="fa fa-angellist fa-lg"></a>
+            <a href="#" class="fa fa-twitter fa-lg"></a>
+            <a href="#" class="fa fa-medium fa-lg"></a>
+        </footer>
+    </div>
+    
+    <!--Works-->
+    <div class="light">
+        <header>
+            <h1>My Recent Works</h1>
+            <hr>
+        </header>
+        <section>
+    
+            <div class="multi-post">
+                <img class="mpimage" src="./assets/multipostplaceholder.png" alt="Place Holder">
+                <div class="post">
+                    <h2>Multi-Post Stories</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                </div>
+            </div>
+    
+            <div class="card-container">
+    
```

Comment:
The multipost looks different in the template the the other cards, and its responsive behavior when the viewport is desktop, acts different too.

Reply:
- [ ] It would be better if you stick the Figma design as the first reviewer said. Your multipost stories card have missing buttons and lists. Please add them under the paragraph. Like same as the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design 🙏.